### ISSUE TITLE
Unable to restore queue state after sudden crash of NZBGet

### ISSUE THREAD

**Issue by PartTheSeas** [nzbget/nzbget#812]
Hey there guys,

sorry to post here, I couldn't find a way to register on the NZBget forum.

My problem is, that NZBGet (v21.2-testing-r2333) crashed all of a sudden on my Windows machine and all me queued items are gone from the Web-UI. However, they're still in the "queue" and "nzb" directory, but they just won't scan again from the Web-UI via "Add + Scan incoming directory".

Prior to that I had an error message right when NZBGet crashed, saying that the disk state is corrupt? I don't think this is the case, as it's a 2 day old NVMe NZBget is running on. I forgot to exclude NZBGet from Windows Security scans, maybe that caused the crash?

Long story short: Can anybody help me recover my queue? There were almost 6000 items in it, which I all added manually for the past couple of days.

Any help will be highly appreciated. If logs are needed tell me which ones you need and I'll provide them right away!

Please help

**Comment by PartTheSeas** [comment]
Now I'm not able to start NZBGet at all
